app/models/user.rb

```diff
@@ -35,6 +35,10 @@ class User < Common::RedisStore
   # mvi 'golden record' data
   attribute :mvi
 
+  # mhv_last_signed_in used to determine whether we need to notify MHV audit logging
+  # This is set to Time.now when any MHV session is first created, and nulled, when logout
+  attribute :mhv_last_signed_in, Common::UTCTime
```

Comment:
Good question. In these scenarios I always find that timestamp is more informative than a boolean. Suppose for example MHV is confused about something we're doing, we can lookup in redis and find the timestamp we have logged, and tell them "hey MHV, check your audit log for a Log in received at 10:30 PM on Monday." With a boolean we don't have that benefit.

It's super simple to change it to boolean, but I think this **could** prove more useful. I do something similar for logged_in and logged_out when working with user sessions and active record.